### PR TITLE
Update UTM params for Twitter

### PR DESCRIPTION
**Pull request by willdurand**
see: https://github.com/mozilla/addons-blog/issues/72#issuecomment-822774586

### CODE REVIEW

File: src/includes/blogpost.njk [hunk 1]
```diff
@@ -24,7 +24,7 @@
       <div class="blogpost-meta-share-twitter">
         <dt class="share-twitter">Share</dt>
         <dd class="share-twitter">
-          <a class="share-twitter-link" href="https://twitter.com/intent/tweet/?text={{ tweetText }}&url={{ tweetURL }}?{{ 'utm_source=twitter&utm_medium=social&utm_campaign=shares_from_blog' | urlencode }}">
+          <a class="share-twitter-link" href="https://twitter.com/intent/tweet/?text={{ tweetText }}&url={{ tweetURL }}?{{ 'utm_source=twitter&utm_medium=social&utm_campaign=amo_blog_share' | urlencode }}">
```
Comment (muffinresearch) [inline src/includes/blogpost.njk:27]: Should UTM params be covered by a test?

**Review by muffinresearch — APPROVED** [review]
r+wc

File: src/includes/blogpost.njk (hunk 1 above)
Comment (willdurand) [reply src/includes/blogpost.njk:27]: Yeah, we could using the Wordpress test suite probably. I am also thinking about having a test suite for the generated content of the blog, but I am not sure how to write that given that content won't be fixed (even knowning the URL of a blogpost is not easy)

Comment (willdurand) [reply src/includes/blogpost.njk:27]: I added a php assertion for now

**Pull request merged**